tests/operators/hive_operator.py

```diff
@@ -163,6 +163,13 @@ def test_hive_stats(self):
                 dag=self.dag)
             t.run(start_date=DEFAULT_DATE, end_date=DEFAULT_DATE, force=True)
 
+        def test_named_hive_partition_sensor(self):
```

Comment:
Added 2 other test to exercise multiple partition and nonexistent partition logic. The order in which things are poked, etc seems like an implementation detail and should not be baked into tests. That poking works is tested in the docstring for `check_for_named_partition`. Let me know if there are other specific tests you think should be in here.